misk-jooq/README.md

```diff
@@ -0,0 +1,93 @@
+## misk-jooq
+
+Seeks to integrate jooq with misk. Jooq is an open source project located here -
+https://www.jooq.org/
+
+In order to use jooq in your service you need to a few things:
+
+1. Add this module as a dependency to your project.
```

Comment:
Would be good to have some code samples of how this module is used (i.e. why it exists).